.buildkite/main-tests.json

```diff
@@ -21,6 +21,10 @@
       "test_name": "style",
       "command": "cargo fmt --all -- --check --config format_code_in_doc_comments=true"
     },
+    {
```

Comment:
LGTM, but I'd add the same also in for staging crates (`.buildkite/staging-tests.json`)